spec/models/room_spec.rb

```diff
@@ -3,5 +3,7 @@
 RSpec.describe Room, type: :model do
   describe "relationships" do
     it { should belong_to :hotel }
+    it { should have_many :room_guests }
+    it { should have_many(:guests).through(:room_guests) }
```

Comment:
Nice work with these model tests. Don't forget to include a model spec for RoomGuests as well! There can be 2 belongs_to assertions there